doc/whatsnew/2.0.rst

```diff
@@ -61,6 +61,10 @@ New checkers
        class MyClass(metaclass=SomeClass):
            pass
 
+* The ``docparams`` extension has a new checker ``missing-raises-doc``, which
```

Comment:
OK. I had added this because as far as I can tell this checker hadn't been documented when it was added. I will remove it if that's your preference.